traffic_portal/app/src/common/modules/form/server/form.server.tpl.html

```diff
@@ -111,8 +111,18 @@
             </div>
 
             <fieldset>
-                <legend>Profiles<button name="addProfileBtn" class="btn btn-primary right-button btn-xs" type="button" title="add a new profile to the server" ng-click="addProfile()"><i class="fa fa-plus"></i></button></legend>
-                <fieldset ng-repeat="profile in server.profileNames track by $index">
+                <legend title="Layered Profiles for a given server."> Profiles
+                    <button name="addProfileBtn" class="btn btn-primary right-button btn-xs" type="button" title="add new profile(s) to the server" ng-click="addProfile()"><i class="fa fa-plus"></i></button>
+                </legend>
+                <div class="helptext">
```

Comment:
This `DIV` doesn't seem to be serving any purpose. It isn't displaying like other `.helptext` elements do - i.e. on hover - and it doesn't appear to be styled in any way so CSS isn't using it. It's just a container element with no semantic meaning, so it's not important to the document structure.

So, if this isn't meant to be on-hover help text, I'd suggest removing the div so nobody seeing that thinks it's broken. If it *is* meant to be on-hover help text, then some work needs to be done to get that to display properly.

Reply:
It is not on-hover help test, so I will remove `<div>`